autora/theorist/bsr/prior.py

```diff
@@ -0,0 +1,175 @@
+from typing import Callable, Dict, Union
+
+import numpy as np
+from scipy.stats import norm
+
+from .misc import normalize_prior_dict
+from .operation import (
+    cos_op,
+    exp_op,
+    inv_op,
+    linear_op,
+    make_pow_op,
+    minus_op,
+    multiply_op,
+    neg_op,
+    plus_op,
+    sin_op,
+)
+
+
+def __get_ops_with_arity():
```

Comment:
Interesting. I created this file with the aim to reach some sort of consistency with the `prior.py` in [BMS](https://github.com/AutoResearch/autora/blob/main/autora/theorist/bms/prior.py) which has been using double underscore so I just followed suit. Do you think I should change it back to a single underscore instead or keep up with the BMS way?

Reply:
Since integration across different theorist is not a priority task at this moment, I will simply follow the pythonic way of using single underscore instead. Thx again for pointing this out!